dropwizard-unixsocket/src/test/java/io/dropwizard/unixsocket/UnixSocketConnectorFactoryTest.java

```diff
@@ -0,0 +1,98 @@
+package io.dropwizard.unixsocket;
+
+import com.codahale.metrics.health.HealthCheck;
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import jnr.unixsocket.UnixSocketAddress;
+import jnr.unixsocket.UnixSocketChannel;
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.Reader;
+import java.nio.channels.Channels;
+
+import static org.junit.Assume.assumeFalse;
+
+public class UnixSocketConnectorFactoryTest {
+
+    @Rule
+    public DropwizardAppRule<Configuration> rule = new DropwizardAppRule<>(HelloWorldApplication.class,
+        ResourceHelpers.resourceFilePath("yaml/usock-server.yml"));
+
+    private File socket = new File("/tmp/dropwizard.sock");
+
+    private String httpRequest = "GET /app/hello HTTP/1.1\r\n" +
+        "Host: dropwizard-unixsock\r\n" +
+        "\r\n";
+
+    @Before
+    public void setUp() {
+        assumeFalse(SystemUtils.IS_OS_WINDOWS);
```

Comment:
Does anyone have an idea why this doesn't seem to work on AppVeyor?

Reply:
By `UnsatisfiedLinkError` looks like that native method not found.
Maybe you have to specify image to run maven in appveyor, e.g. `image: ubuntu`

Here more info:
https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/#running-your-build-on-linux

Reply:
@alex-shpak The whole point of using AppVeyor is to make sure that Dropwizard can be built on Windows. 😉

From my understanding, the assumption in the `@Before` method should prevent all tests from running on Windows. But that doesn't seem to work. 🤔

* https://static.javadoc.io/junit/junit/4.12/org/junit/Assume.html
* https://github.com/junit-team/junit4/wiki/assumptions-with-assume

Reply:
Try:
```java
    @BeforeClass
    public static void setUp() {
        assumeFalse(SystemUtils.IS_OS_WINDOWS);
    }
```

Reply:
@nickbabcock Seems to have worked. Thanks!
https://ci.appveyor.com/project/arteam/dropwizard/build/1838#L8658